fsharp-backend/tests/FuzzTests/HttpClient.FuzzTests.fs

```diff
@@ -82,7 +82,7 @@ module Tests =
     let test name fn = testProperty config typeof<Generator> name fn
     testList
       "HttpClient, known good"
-      [ test "dvalToUrlStringExn" dvalToUrlStringExn // FSTODO: unicode
+      [ test "dvalToUrlStringExn" dvalToUrlStringExn // FSTODO: unicode WHATISTHIS
```

Comment:
What's the same expectation here? is it the same expectation as in Generators.fs?

Reply:
Yes, I think so